src/OrbitCore/CoreUtils.h

```diff
@@ -33,10 +32,6 @@
 #include <utility>
 #include <vector>
 
-inline uint64_t StringHash(std::string_view str) {
-  return XXH64(str.data(), str.size(), 0xBADDCAFEDEAD10CC);
```

Comment:
Was this also the last usage of xxHash64 in OrbitCore? If yes, could you also remove it from the dependencies in `CMakeLists.txt`?

Reply:
I believe presets still use it - old format and conversion function.

Reply:
It's used in `ClientData` but I can drop it from `OrbitCore?CMakeLists.txt`.

Reply:
This is done.